domain-models-runtime/src/test/java/org/folio/rest/tools/utils/TenantLoadingTest.java

```diff
@@ -268,15 +268,36 @@ public void testPutFailure(TestContext context) {
 
     TenantLoading tl = new TenantLoading();
     tl.addJsonIdContent("loadRef", "tenant-load-ref", "data", "data");
-    putStatus = 422;
+    putStatus = 423;
     tl.perform(tenantAttributes, headers, vertx, res -> {
       context.assertTrue(res.failed());
       async.complete();
     });
   }
 
   @Test
-  public void testOkStatusCode(TestContext context) {
+  public void testokWithAcceptStatus(TestContext context) {
+    Async async = context.async();
+    List<Parameter> parameters = new LinkedList<>();
+    parameters.add(new Parameter().withKey("loadRef").withValue("true"));
+    TenantAttributes tenantAttributes = new TenantAttributes()
+      .withModuleTo("mod-1.0.0")
+      .withParameters(parameters);
+    Map<String, String> headers = new HashMap<String, String>();
+    headers.put("X-Okapi-Url-to", "http://localhost:" + Integer.toString(port));
+
+    TenantLoading tl = new TenantLoading();
+    tl.addJsonIdContent("loadRef", "tenant-load-ref", "data", "data");
+    putStatus = 423;
+    tl.withAcceptStatus(423);
+    tl.perform(tenantAttributes, headers, vertx, res -> {
+      context.assertTrue(res.succeeded());
+      async.complete();
+    });
```

Comment:
`Async async` is not needed:
tl.perform(tenantAttributes, headers, vertx, context.asyncAssertSuccess())